src/sedtrails/particle_tracer/particle.py

```diff
@@ -16,52 +16,57 @@ class Time:
 
     Attributes
     ----------
-    time_step : int
-        The current time step in a simulation. A non-negative integer.
+    reference_date : numpy.datetime64
```

Comment:
Could you modify the datatype of `reference_date` so that we pass a string with the date, instead of the numpy object? For example: `reference_date= "2025-05-14"`

Reply:
Then, the class does the transformation that are needed.